src/tools/connection_patcher/Program.cpp

```diff
@@ -32,6 +32,9 @@
 
 #if PLATFORM == PLATFORM_WINDOWS
 #include <Shlobj.h>
+#elif PLATFORM == PLATFORM_UNIX
```

Comment:
I don't own a MAC to test, but what I can tell is that PLATFORM_LINUX is returning false in my machine (Linux)